.travis.yml

```diff
@@ -23,7 +23,7 @@ before_install:
   - zsh  --version
   - dpkg -s dash | grep ^Version | awk '{print $2}'
 install:
-  - if [ -z "${SHELLCHECK-}" ]; then nvm install node && npm install && npm prune && npm ls urchin doctoc eclint dockerfile_lint; fi
+  - if [ -z "${SHELLCHECK-}" ]; then nvm install --lts && npm install && npm prune && npm ls urchin doctoc eclint dockerfile_lint; fi
```

Comment:
why would node 18 fail but node 16 pass?

if so, this would start breaking again as soon as 18 becomes LTS.

Reply:
Node 18 needs a newer version of GLIBC than what's available on xenial.
```
node: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.27' not found (required by node)
node: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.25' not found (required by node)
node: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by node)
```